api_docs/charts.json

```diff
@@ -8,6 +8,7 @@
         "type": "Function",
         "children": [
           {
+            "id": "def-public.ColorPicker.{\n- onChange,\n  color: selectedColor,\n  label,\n  useLegacyColors = true,\n  colorIsOverwritten = true,\n  onKeyDown,\n}",
```

Comment:
I don't think this is the intended key or label here?

Reply:
Well, it actually is correct because it's a destructured function parameter. 

If you have:
```
const myfn = (prop: SomeType) => string
```
The id for the parameter is `def-public.myfun.prop`.

But if it's written like this:
```
const myfn = ({ foo, bar, zed }: SomeType) => string
```

The id for the parameter is `def-public.myfun.{-foo,-bar,-zed-}`.

I can run these ids through some cleanup though so it looks a lot better:

<img width="832" alt="Screen Shot 2021-04-06 at 5 26 25 PM" src="https://user-images.githubusercontent.com/16563603/113780516-51e84180-96fd-11eb-8a3c-d9a779139db4.png">

Reply:
I wonder if we just want to refer to them positionally or something, like `def-public.myfun.$1` maybe

Reply:
sure, I can do that.